docs/input-data/crac/json.md

```diff
@@ -383,7 +383,7 @@ crac.newFlowCnec()
     "unit" : "megawatt",
     "min" : -1500.0,
     "max" : 1500.0,
-    "side" : "left"
+    "side" : "ONE"
```

Comment:
if you accept my proposal above, this shall look like this:
```suggestion
    "side" : 1
```

Reply:
done